_dev/src/main.ts

```diff
@@ -50,6 +53,9 @@ new Vue({
           version_ps: this.$store.state.app.psVersion,
           ggl_module_version: this.$store.state.app.psxMktgWithGoogleModuleVersion,
           ggl_api_endpoint: this.$store.state.app.psxMktgWithGoogleApiUrl,
+          ggl_user_has_enabled_campaign: isActiveCamp,
+          ggl_user_has_googleAds_connected: !!this.$store.state.googleAds.accountChosen,
+          ggl_user_has_productFeed_exported: this.$store.state.productFeed.isConfigured,
```

Comment:
Did you manage to test it?

Reply:
In order to check the data is properly sent to Hotjar, building the app with `npm run build` should allow the tracking to run

Reply:
As it is an identify, it is suppose to be sent to the attributes in hotjar filters. However we are not able to test it without deploying it
![Capture d’écran 2022-04-04 à 10 22 12](https://user-images.githubusercontent.com/37299291/161503990-5821db88-a5b8-4cc8-b170-447ecd791401.png)

![Capture d’écran 2022-04-04 à 10 36 34](https://user-images.githubusercontent.com/37299291/161506208-01dd304d-dc4f-405e-ad23-8017f768e8a2.png)